index.html

```diff
@@ -6,7 +6,411 @@
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button" type="button">
+              <span class="visually-hidden">1 фото</span>
+            </button>
+            <p class="slider-description">
+              <b class="slider-description-title">
+                Порхает как бабочка,<br>жалит как пчела!
+              </b>
+            </p>
+            <p class="slider-description">
+              Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.
+            </p>
+            <p class="slider-paragraph">
+              <a class="slider-link" href="#">Подробнее</a>
+            </p>
+            <p class="slider-paragraph">
+            <table class="slider-table">
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    800 м
+                  </b>
+                </td>
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    50 м
+                  </b>
+                </td>
+              </tr>
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <span class="slider-table-cell-title">
+                    Дальность полета
+                  </span>
+                </td>
+                <td class="slider-table-item">
+                  <span class="slider-table-cell-title">
+                    Радиус поражения
+                  </span>
+                </td>
+              </tr>
+            </table>
+            </p>
+          </li>
+
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button" type="button">
+              <span class="visually-hidden">2 фото</span>
+            </button>
+            <p class="slider-description">
+              <b class="slider-description-title">
+                Худеем <br>правильно!
+              </b>
+            </p>
+            <p class="slider-description">
+              Мотивирующие фитнес-браслеты помогут найти в себе силы не пропускать занятия и соблюдать диету.
+            </p>
+            <p class="slider-paragraph">
+              <a class="slider-link" href="#">Подробнее</a>
+            </p>
+            <p class="slider-paragraph">
+            <table class="slider-table">
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    48 часов
+                  </b>
+                </td>
+              </tr>
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <span class="slider-table-cell-title">
+                    Без подзарядки
+                  </span>
+                </td>
+              </tr>
```

Comment:
1. Не надо заниматься стилизацией тэгом `b`. Стилизацией вы занимаетесь потом2
2. Думай про семантику. Обычно название категорий в шапке таблицы
3. То что категории сверху пока не обращай внимание, как и то что они полужирным шрифтом

Взял для примера самую маленькую таблицу, для остальных тоже самое
```suggestion
              <thead>
                <tr class="slider-table-row">
                  <th class="slider-table-cell-title">Без подзарядки<th>
                </tr>
              </thead>
              <tbody>
                <tr class="slider-table-row">
                  <td class="slider-table-cell-info">48 часов</td>
                </tr>
              </tbody>
```

Reply:
На 125 у вас th не закрыт, если я правильно понимаю?

Reply:
Не вижу, но я мне и неудобно писать в комментах